ui/src/app/components/popups.tsx

```diff
@@ -0,0 +1,285 @@
+import {
+  clamp,
+  flow,
+  isEqual,
+  pick,
+  uniqueId
+} from 'lodash/fp';
```

Comment:
Note: we likely can configure our build to just have a `_` global. I'm not sure whether I support doing that, but something to consider.

Reply:
I actually argued against that, and moved to this, because I didn't like the convention for the sake of understandability to people not familiar with Lodash. I might be okay with having a `lodash` global? But `_` I don't think is very clear to people without the background.